src/Slider.jsx

```diff
@@ -4,7 +4,14 @@ import linear from './algorithms/linear';
 
 // istanbul ignore next
 function getDOMNode(node) {
-  return React.version.indexOf('0.14') > -1 ? node : node.getDOMNode();
+  // Fix for version 15.0.2
+  var version = _react2['default'].version;
```

Comment:
`['default']` is the kind of thing that belongs in babel output. I think you mean `React.version` here.